src/components/spaces/canvas/index.js

```diff
@@ -222,10 +222,16 @@ export default class Canvas extends Component{
 
       let pathStatus = unconnectedStatus
       if (!withinSelectedRegion && (outputIsAncestor || inputIsDescendant)) {
+        const degree = outputIsAncestor ?
+          ancestors.findIndex(e => e.id === outputId) + 1
+          :
+          descendants.findIndex(e => e.id === inputId) + 1
+
         pathStatus = outputIsAncestor ? 'ancestor' : 'descendant'
+        if (degree === 1) { pathStatus += '-1-degree' }
       }
 
-      return {input, output, pathStatus, hasErrors: hasErrors(simulation) }
+      return {input, output, pathStatus, hasErrors: hasErrors(simulation), className: '' }
```

Comment:
Seems like you added code to deal with you setting a `className` in other files, but then always set it to `''`? Am I missing something, or can some of this stuff go?